xbmc/guilib/DDSImage.cpp

```diff
@@ -120,12 +119,12 @@ bool CDDSImage::Create(const std::string &outputFile, unsigned int width, unsign
 {
   if (!brga)
     return false;
-  if (!Compress(width, height, pitch, brga, maxMSE))
-  { // use ARGB
-    Allocate(width, height, XB_FMT_A8R8G8B8);
-    for (unsigned int i = 0; i < height; i++)
-      memcpy(m_data + i * width * 4, brga + i * pitch, std::min(width * 4, pitch));
-  }
+
+  // use ARGB
+  Allocate(width, height, XB_FMT_A8R8G8B8);
+  for (unsigned int i = 0; i < height; i++)
+    memcpy(m_data + i * width * 4, brga + i * pitch, std::min(width * 4, pitch));
+
```

Comment:
yep. it's an advanced setting for dds fanart. I am just the "clean up monkey", so I am not sure if I understand all this code, but I'll see what I can do. thanks for the feedback.